model/kestrel/metrics.yaml

```
@@ -38,14 +38,12 @@ groups:
       - ref: network.protocol.version
         examples: ["1.1", "2"]
       - ref: tls.protocol.version
+      # yamllint disable rule:line-length
       - ref: error.type
         brief: The full name of exception type.
         requirement_level:
           conditionally_required: if and only if an error has occurred.
-        note: "Captures the exception type when a connection fails."
         examples: ['System.OperationCanceledException', 'Contoso.MyException']
```

(using the examples from below)
```suggestion
        examples: ['connection_reset', 'invalid_handshake']
```